src/NServiceBus.Persistence.InMemory.AcceptanceTests/.editorconfig

```diff
@@ -4,4 +4,7 @@
 dotnet_diagnostic.CA1711.severity = none
 
 # Justification: Test project
-dotnet_diagnostic.CA2007.severity = none
\ No newline at end of file
+dotnet_diagnostic.CA2007.severity = none
+
+# IDE0028: Simplify collection initialization
```

Comment:
**Nitpick**: Can you please update this comment to have the same format as the others, meaning: "Justification:..."